packages/plugin-ext/src/plugin/status-bar/status-bar-item.ts

```diff
@@ -140,4 +133,8 @@ export class StatusBarItemImpl implements theia.StatusBarItem {
     public dispose(): void {
         this.hide();
     }
+
+    static nextId(): string {
+        return 'plugin-status-bar-item:' + UUID.uuid4();
```

Comment:
please keep the constant variable
const STATUS_BAR_MESSAGE_PRE = 'status-bar-entry';